js/apps/system/_admin/aardvark/APP/react/src/components/jsonEditor/JSONErrors.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from "@chakra-ui/react";
+import { ValidationError } from "jsoneditor-react";
+import React from "react";
+
+export const JSONErrors = ({ errors }: { errors?: ValidationError[] }) => {
+  if (!errors || errors.length === 0) {
+    return null;
+  }
+  return (
+    <Box
+      maxHeight={"80px"}
+      paddingX="2"
+      paddingY="1"
+      fontSize="sm"
+      color="red"
+      background="red.100"
+      overflow={"auto"}
+    >
+      {errors.map(error => {
+        return (
+          <Box>{`${error.keyword} error: ${
```

Comment:
It's not asking for a `key` prop in console? 🤔

Reply:
Oh yes, this component was already there and I missed it. Thanks!